apex/contrib/csrc/fmha/fmha_api.cpp

```diff
@@ -0,0 +1,281 @@
+/***************************************************************************************************
+ * Copyright (c) 2011-2021, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification, are not permit-
+ * ted.
```

Comment:
This is not an appropriate license for an open-source project; please fix to match APEX -- but also please ensure that doing so is permissible based on the original source of the code being contributed.

(Same comment for all files.)